### DISCUSSION TITLE
Gstremer is not connecting to stream in AWS Kinesis

### DISCUSSION THREAD

**Discussion by ambroseeverhart** [awslabs/amazon-kinesis-video-streams-webrtc-sdk-c#1885] [Q&A]
Hello Smart folks,
I want to know how  stream video through AWS Kinesis with kvsWebrtcClientMaster with rtsp as a video source. Thanks in advance for any suggestions and helps .

**Comment by stefankiesz** [comment]
Hi, there is a Master sample using GStreamer with RTSP source [here](https://github.com/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/blob/f1073439213db4aa4b4b7cc1423609d920dd5489/samples/kvsWebrtcClientMasterGstSample.c#L175).
